src/pages/collection/Collection.tsx

```diff
@@ -84,6 +94,8 @@ function Collection() {
   const [showDisplaySettingsModal, setShowDisplaySettingsModal] = useState(false);
   const [groupTotal, setGroupTotal] = useState(0);
   const [timelineSeries, setTimelineSeries] = useState<SeriesType[]>([]);
+  const [search, setSearch] = useState('');
+  const debouncedSearch = useDebounce(search, 200);
```

Comment:
should we add some rule to only use hook for debounce? I think we use lodash debounce in other places.